lib/vendor.rb

```diff
@@ -1,3 +1,24 @@
 class Vendor
+
+    attr_reader :name,
+                :inventory
+
+    def initialize(name)
+        @name = name
+        @inventory = Hash.new { |hash, key| hash[key] = { quantity: 0 } }
+    end
   
+    def check_stock(item)
+        @inventory[item][:quantity]
+    end
+
+    def stock(item, quantity)
+        @inventory[item][:quantity] += quantity
```

Comment:
Was expecting something a little more like this here `@inventory[item] += quantity`